packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -253,17 +253,18 @@ class CupertinoPageRoute<T> extends PageRoute<T> {
 
   @override
   Widget buildPage(BuildContext context, Animation<double> animation, Animation<double> secondaryAnimation) {
+    final Widget child = builder(context);
     final Widget result = Semantics(
       scopesRoute: true,
       explicitChildNodes: true,
-      child: builder(context),
+      child: child,
```

Comment:
This looks like a perfectly OK change. Is there an issue with the code crashing before it gets to the assert?

Reply:
I was not aware of an open issue mentioning that, but I tried earlier, it doesn't crash, but it shows a black empty screen instead of ErrorWidget with the red screen. 
If this looks like a perfectly ok change, can I merge this one?

Reply:
SG!